src/StripStylesTransformer.ts

```diff
@@ -43,24 +46,11 @@ const COMPONENT = 'Component';
 /** All props to be transformed inside a decorator */
 const TRANSFORM_IN_DECORATOR_PROPS = [STYLES];
 
-/**
- * Transformer ID
- * @internal
- */
-export const name = 'angular-component-strip-styles';
-
-// increment this each time the code is modified
-/**
- * Transformer Version
- * @internal
- */
-export const version = 1;
-
 /**
  * The factory of hoisting transformer factory
  * @internal
  */
-export function factory(cs: ConfigSet) {
+export function factory(cs: ConfigSet & { compilerModule: TTypeScript }) {
```

Comment:
This factory type isn’t the same as the one in AngularConstructorParametersTransformer.ts. Is it intended ?